data/categories/sg_sporting_goods.yml

```diff
@@ -12382,24 +12382,126 @@
   children:
   - sg-4-15-1
   - sg-4-15-2
+  - sg-4-15-3
   attributes:
   - age_group
   - color
   - pattern
 - id: sg-4-15-1
   name: E-Scooters
+  children:
+  - sg-4-15-1-1
+  - sg-4-15-1-2
+  - sg-4-15-1-3
+  - sg-4-15-1-4
+  attributes:
+  - age_group
+  - color
+  - pattern
+- id: sg-4-15-1-1
+  name: Commuter E-Scooters
   children: []
   attributes:
   - age_group
   - color
```

Comment:
We are creating a separate category for `Youth E-Scooters`; I wouldn't add the attribute `age group` for things like Commuter E-Scooters.